float64[] robTool_tcp
string vision_ip
int16 port
int16 calibrate_num
---
bool is_success
string info